pkg/sql/catalog/descs/descriptor.go

```diff
@@ -263,11 +264,14 @@ func getSchemaByName(
 		return true, sc, nil
 	}
 	if isTemporarySchema(name) {
-		if refuseFurtherLookup, sc, err := tc.temporary.getSchemaByName(
-			ctx, txn, db.GetID(), name, tc.settings.Version,
-		); refuseFurtherLookup || sc != nil || err != nil {
```

Comment:
`refuseFurtherLookup` is always `true` with the current implementation of `tc.temporary.getSchemaByName`. I hoisted part of it in here, I thought it was clearer that way.